test_output/help/server_delete.txt

```diff
@@ -22,6 +23,7 @@ knife bmcs server delete (options)
     -z, --local-mode                 Point knife commands at local repository instead of server
     -u, --user USER                  API Client Username
         --print-after                Show the data after a destructive operation
+        --purge                      Also remove node from Chef server. Chef node name defaults to the instance display name unless node-name is specified.
```

Comment:
Remove the corresponding node from the Chef Server. The instance display name will be used as the node name, unless --node-name is specified.